add-scene-layer-with-elevation/src/main/java/com/esri/arcgismaps/sample/addscenelayerwithelevation/components/SceneViewModel.kt

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.addscenelayerwithelevation.components
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.layers.ArcGISSceneLayer
+import com.arcgismaps.mapping.view.Camera
+import com.esri.arcgismaps.sample.addscenelayerwithelevation.R
+
+class SceneViewModel(private val application: Application) : AndroidViewModel(application) {
+    // set the SceneView state
+    val sceneViewState = SceneViewState()
+
+    // add a scene layer to the ArcGISScene
+    init {
+        // create a scene layer from a scene service for viewing buildings
+        var sceneLayer =  ArcGISSceneLayer(application.getString(R.string.brest_buildings))
```

Comment:
This doesn't need to be a `var` type. 
```suggestion
        val sceneLayer =  ArcGISSceneLayer(application.getString(R.string.brest_buildings))
```